docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -1210,6 +1213,65 @@ For more information on the relationship between the elements, see the [element
 <feature name="http://tizen.org/feature/fmradio">true</feature>
 ```
 
+### \<dependencies\> Element
+
+This element contains a set of packages that have dependencies on Tizen package. The package can be installed when all dependencies are satisfied.
```

Comment:
@jeremy-jang Kindly replace "Tizen package" with "Tizen packages". While generalizing "Tizen packages" is an accurate way of putting it.

Reply:
@rishu-samsung At the starting point of this page, we can see this sentence.

>**A Tizen native application package** contains a manifest file, called tizen-manifest.xml, which is used to describe the application information.  ...
>... The following sections summarize the elements used in the tizen-manifest.xml file of a native application. ...

This overall page is to describe "A Tizen Native Application package's manifest file".
As every section is describing parts of "A Tizen Native Application package's manifest file", please consider the context also while you review the update.

To my understand, 
a Tizen Native Application package can have dependencies, and the package can be installed only when the dependencies are satisfied.
To list up the required dependencies information, we have `<dependency> Element`.

In this point of view, I think your suggestion here is wrong, as the "Tizen package" is not a word to show generalized packages but a specific one "this" package that we are examining as written in the starting point of this page.

**Original sentence :** This element contains a set of packages that have dependencies to this Tizen package. The package can be installed when all dependencies are satisfied.

I'm worrying about the case  that the meaning is changed wrongly during the review, so hope you to check the comment again.

Reply:
@mijin85cho 
I understand your concern. This was the same question in my previous comment, https://github.com/Samsung/tizen-docs/pull/1143#discussion_r44067761.
@jeremy-jang confirmed that there is no specific package. Hence, I agreed to generalize.

Reply:
Basically the package can be installed only if the dependencies described at the packages's manifest are satisfied. So I used "this Tizen package" at first. (as @mijin85cho said)
I changed to the generalized word as @rishu-samsung reviewed, because i'm not sure that using "this Tizen package" because i cannot find word such as "this Tizen package" or "this Tizen application" in this page. Which word is more proper?

Reply:
@mijin85cho I completely understand your concern. In my previous comment, I have asked a question regarding the same, https://github.com/Samsung/tizen-docs/pull/1143#discussion_r440677611. 

As @jeremy-jang clarified that it is generalized, I agreed to that.

However, now I understand that it is not a generalized package. It is “Tizen Native Application package”. As the file is long and usage of a pronoun for a heading can create confusion, we should specify the package to make it clear as:

### \<dependencies\> Element 
This element contains a set of packages that have dependencies on Tizen Native Application package.

For example, in this page, the description for other elements are clear 
<manifest> Element

This element contains the manifest information for a Tizen native application.

Reply:
@rishu-samsung Or, I think "the Tizen package" would be also good. As this page is under 
native-tools category, it is very clear that this document is about Tizen Native. We don't need to emphasize "Tizen Native" at everywhere...

### \<dependencies\> Element

This element contains a set of packages that have dependencies on the Tizen package. The package can be installed when all dependencies are satisfied.

Would you let me know what your opinion is?

Reply:
@mijin85cho It is good to be specific and mention which package.
However, as you explained, it is very clear that this document is about Tizen Native and if we don't need to emphasize "Tizen Native" everywhere, then I will go with your suggestion.

@jeremy-jang Please update as suggested by @mijin85cho

Reply:
I updated it. Thanks.